src/python/pants/init/parsed_specs_calculator.py

```diff
@@ -76,18 +91,20 @@ def create(
     # Determine the `--owner-of=` arguments provided from the global options
     owned_files = options.for_global_scope().owner_of
 
-    logger.debug('address_spec_roots are: %s', address_spec_roots)
+    logger.debug('parsed_specs are: %s', parsed_specs)
     logger.debug('changed_request is: %s', changed_request)
     logger.debug('owned_files are: %s', owned_files)
-    targets_specified = sum(1 for item
-                         in (changed_request.is_actionable(), owned_files, address_spec_roots.dependencies)
-                         if item)
+    targets_specified = sum(
+      1 for item
+      in (changed_request.is_actionable(), owned_files, parsed_specs.provided_specs.dependencies)
```

Comment:
NB: the constructor for `ParsedSpecs` will have already validated that the user didn't try to specify both filesystem specs and address specs. We only need to check whether any specs were given at the same time as `--changed-*` and `--owner-of`.